client/rpc-core/src/types/transaction.rs

```diff
@@ -51,114 +54,110 @@ pub struct Transaction {
 	/// The miner's tip.
 	#[serde(skip_serializing_if = "Option::is_none")]
 	pub max_priority_fee_per_gas: Option<U256>,
-	/// Gas
-	pub gas: U256,
 	/// Data
 	pub input: Bytes,
 	/// Creates contract
+	#[serde(skip_serializing_if = "Option::is_none")]
 	pub creates: Option<H160>,
-	/// Raw transaction data
-	pub raw: Bytes,
-	/// Public key of the signer.
-	pub public_key: Option<H512>,
 	/// The network id of the transaction, if any.
+	#[serde(skip_serializing_if = "Option::is_none")]
 	pub chain_id: Option<U64>,
-	/// The standardised V field of the signature (0 or 1).
-	pub standard_v: U256,
+	/// Pre-pay to warm storage access.
+	#[serde(skip_serializing_if = "Option::is_none")]
+	pub access_list: Option<Vec<AccessListItem>>,
+	/// The parity (0 for even, 1 for odd) of the y-value of the secp256k1 signature.
+	#[serde(skip_serializing_if = "Option::is_none")]
+	pub y_parity: Option<U256>,
 	/// The standardised V field of the signature.
-	pub v: U256,
+	///
+	/// For backwards compatibility, `v` is optionally provided as an alternative to `yParity`.
+	/// This field is DEPRECATED and all use of it should migrate to `yParity`.
+	#[serde(skip_serializing_if = "Option::is_none")]
+	pub v: Option<U256>,
 	/// The R field of the signature.
 	pub r: U256,
 	/// The S field of the signature.
 	pub s: U256,
-	/// Pre-pay to warm storage access.
-	#[cfg_attr(feature = "std", serde(skip_serializing_if = "Option::is_none"))]
-	pub access_list: Option<Vec<AccessListItem>>,
```

Comment:
I don't know why we use `#[cfg_attr(feature = "std", ...)` here, maybe it's a error caused `copy c + v`